src/Workspaces/Remote/ServiceHub/ExternalAccess/Razor/Api/TestHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Api
+{
+    internal static class TestHelpers
+    {
+        /// <summary>
+        /// Allows tests to create a new remote workspace, rather than use RemoteWorkspaceManager.Default, so things can be isolated
+        /// </summary>
+        public static Workspace CreateTestWorkspace()
+            => new RemoteWorkspaceManager(workspace => new SolutionAssetCache(workspace, cleanupInterval: TimeSpan.FromSeconds(30), purgeAfter: TimeSpan.FromMinutes(1))).GetWorkspace();
```

Comment:
don't love these hardcoded lceanups.

Reply:
If it makes you feel any better, they're never used, and nor is the solution asset cache, in our tests anyway.

Reply:
On second thought, I'm just going to delete this file. I didn't end up needing it now, so better to leave it for later